Packages/manifest.json

```diff
@@ -27,6 +27,7 @@
     "com.unity.xr.management": "4.3.3",
     "com.unity.xr.oculus": "3.3.0",
     "com.unity.xr.openxr": "1.8.2",
+    "com.unity.xr.openxr.picoxr": "https://github.com/icosa-mirror/com.unity.xr.openxr.picoxr.git#1.2.0-openbrush",
```

Comment:
Does this affect the other builds? Does it need to be removed for them? (I hope not)

Reply:
Hopefully not! we have to use a mini branch for the same reason we delete the meta file on the other Pico plugin - these clashes are probably why it fails on this CI run.

Reply:
Can't be a clash; the file clearly has a unique copyright! ;-)